### PR TITLE
Create w3id link for OFO

### PR DESCRIPTION
**Pull request by AlexDonkers**

### CODE REVIEW

File: ofo/.htaccess [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+RewriteRule ^$ https://alexdonkers.github.io/ofo/406.html [R=406,L]
+
+# Default response
+# ---------------------------
+# Rewrite rule to serve the RDF/XML content from the vocabulary URI by default
+RewriteRule ^$ https://alexdonkers.github.io/ofo/ [R=303,L]
+
+
+# If suffix html, redirect to html version
+RewriteRule ^ofo.html$ https://alexdonkers.github.io/ofo/ [R=303,L]
```
Comment (dgarijo) [inline ofo/.htaccess:51]: This is creating a rule for w3id.org/ofo/ofo.html. Is this what you intend?

Comment (AlexDonkers) [reply ofo/.htaccess:51]: Hi, I'm sorry, this rule shouldn't be there. I'll delete it.

Comment (dgarijo) [reply ofo/.htaccess]: Same thing happens for the other serializations: ofo.ttl, jsonld, etc. Is this also an error?

Comment (AlexDonkers) [reply ofo/.htaccess]: Just deleted all these similar rules. Hope the file is fine now.

**Pull request merged**